definitions/divorce/json/CaseTypeTab/CaseTypeTab-general-referral-nonprod.json

```diff
@@ -6,9 +6,19 @@
     "TabID": "generalReferral",
     "TabLabel": "General Referral",
     "TabDisplayOrder": 24,
-    "CaseFieldID": "GeneralReferralReason",
+    "CaseFieldID": "GeneralReferrals",
     "TabFieldDisplayOrder": 1
   },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
```

Comment:
What's this "channel" field?

Reply:
I'm not sure. All the tab objects have it, but not entirely sure what it is used for.

Reply:
What does the story ask of the tab?